lighthouse-viewer/gulpfile.js

```diff
@@ -83,7 +83,6 @@ gulp.task('browserify', () => {
           removeComments: true
         })
         .transform('brfs')
-        .ignore('../lighthouse-core/lib/log.js')
```

Comment:
> think viewer still wants to omit logger from the brfs output.

FWIW, this is just a bundle-size optimization that won't matter after viewer switches to LH2 in #2521